blobstore-fs/src/main.rs

```diff
@@ -8,12 +8,13 @@ use std::time::SystemTime;
 #[allow(unused_imports)]
 use std::{
     collections::HashMap,
+    fs::File,
```

Comment:
Can you please swap this out for `tokio::fs::{File, FileOptions}` as well? The std lib stuff is all sync

Reply:
Ah thanks for pointing this out!

- [ ] remove import for `tokio::fs`